src/components/token-screen-content-container/token-screen-content-container.tsx

```diff
@@ -16,14 +21,24 @@ const historyComponentIndex = 0;
 
 export const TokenScreenContentContainer: FC<Props> = ({ historyComponent, infoComponent }) => {
   const styles = useTokenScreenContentContainerStyles();
+  const { navigate } = useNavigation();
+  const [, isBakerSelected] = useSelectedBakerSelector();
```

Comment:
remove this comma

Reply:
i need second element of array of that selector, so its 100% legal operation